src/typeahead/typeahead.js

```diff
@@ -499,6 +499,19 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position'])
     };
   }])
 
+  .directive('focusMe', function(){
+     return {
+       restrict: 'A',
+       link: function(scope,element,attrs){
+         scope.$watch(attrs.shouldFocus,function(newValue,oldValue){
+           if (newValue) { 
+            element[0].scrollIntoView(false);
```

Comment:
This does not appear to be a valid solution: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#Browser_compatibility